### ISSUE TITLE
⚠ steamtinkerlaunch wiki discussion/contributing

### ISSUE THREAD

**Issue by zany130** [sonic2kk/steamtinkerlaunch#457]
Have some free time so I'm correctly going through the wiki to make some things clearer or adding relevant links. 

Also as we found out on in several  old issues [such as here](https://github.com/frostworx/steamtinkerlaunch/issues/378) gamemode can break a few options such as GDB and cheat engine so I would like to add notes about that. BTW do you know why so i can add it to the wiki? I'm assuming it's a limitation on how gamemode works 

anyway take a look here for my changes whenever you want https://github.com/zany130/steamtinkerlaunch/wiki/ 

going to have to figure out how I can merge these back into your wiki since I don't think github has pull requests for wikis 😅

**Comment by zany130** [comment]
made a few changes [here](https://github.com/zany130/steamtinkerlaunch/wiki/Wine-Debug) and [here](https://github.com/zany130/steamtinkerlaunch/wiki/Steam-First-Time-Setup) to add a warning about an issue I have encountered where if you use steamtinkerlaunch to delete the proton prefix, steam will overwrite steam cloud saves for that game(mentioned the backup feature though and that it does that by default)

some additions to the [gamemode](https://github.com/zany130/steamtinkerlaunch/wiki/GameMode) page and examples of what you can do with it (especial the scripting function which I think is really nice)

some grammar and spell checking in the [cheatengine](https://github.com/zany130/steamtinkerlaunch/wiki/Cheat-Engine) page as well as a note on my experience with some cheat tables which are now "protected" by a new app (should prob mention that fearless revolution is the main source for cheat tables and worked with upstream I think) and that was all I think, for now at least

EDIT: Oh forgot to mention I also added a link to the WineHQ debug chanels page to [Wine Debug](https://github.com/zany130/steamtinkerlaunch/wiki/Wine-Debug)

also any critic (such as if the above warning was necessary to add) or comments would greatly be appreciated

**Comment by frostworx** [comment]
Oh nice! Thank you very much for improving the wiki! :)
You forgot a "n" in channels in your [Wine Debug note](https://github.com/zany130/steamtinkerlaunch/wiki/Wine-Debug), but apart from that I haven't found anything which should be changed.
Just invited you as collaborator, so you can make the changes directly in the wiki. :)

**Comment by zany130** [comment]
Ok updated your wiki with my changes. I also added the steam cloud warning here https://github.com/frostworx/steamtinkerlaunch/wiki/Compatdata-Delete. What do you think of it ? is the warning unnecessary (especially on 3 different but related wiki pages)? Is it clear?

 Should we leave this issue open to communicate feedback on wiki changes? Maybe we can even rename it to make it a general issue for anyone who wants to contribute to the wiki can post ideas?

**Comment by frostworx** [comment]
Thanks a lot for the updates! I do not think the warning is unnecessary _(steam removes compatdata dirs when uninstalling a game, which should have a warning as well :))_
Good idea to make this a generic wiki discussion/contributing issue! 👍
I'll also make it sticky.

**Comment by sonic2kk** [comment]
In my time fiddling around with MO2 and a few games using it (Skyrim SE, TES Oblivion, and FONV) it's been mostly smooth but there is a bit of a recurring issue I've noticed with regards to a missing `Temp` folder when modding Oblivion and FONV. More information is available here: rockerbacon/modorganizer2-linux-installer#316

Basically boils down to this:

- MO2 wants a folder called `Temp` in the prefix it launches the game from, whether that be the compatdata prefix for the game in the Steam Library Folder, or the prefix in `~/.config/steamtinkerlaunch/mo2/compatdata/pfx`
- This folder seems to (correctly) always exist and get used when the game is launched from the MO2 option on the main menu (the STL config directory prefix)
- Sometimes when the game is launched with MO2 using the MO2 GUI option (the Steam Library Folder prefix) to start the game (basically let the game time out and open MO2 on its own, without using the main menu) the prefix for the game in `/path/to/steamapps/compatdata/<appid>/pfx/drive_c/users/steamuser` has this `Temp` folder, sometimes it gets lost and MO2 gets very upset about this, telling you that it can't update/save the mod list.
- This is not too much of an issue as you can still launch the game and use your mods, but installing/configuring mods doesn't really work when its in this state. LOOT also does not work (pretty important)
- When MO2 is in this state it brings up two annoying dialog boxes telling you it can't update/write the mods list every time you open it, try to configure/install some mods, or when you close it.

I'd like to add a note about this to the wiki, worded a little better than I did above perhaps, but before I do I'd like to ask: Is this something STL could potentially work around? Could it create this folder if it doesn't exist when MO2 is launched? You could make the (valid) argument that the MO2 Main Menu option should be used for installing and configuring your mods, and you should only use MO2 to launch your game when you use the MO2 GUI option during a game's launch. But the problem is, it's very unfriendly to a user to get spammed with these MO2 error prompts.

If it's not something STL can fix or something that won't be fixed in the near-future, I can add a note about it on the wiki. Or maybe it should be added anyway, as a "just-in-case" scenario :)

EDIT: It seems like I can reliably reproduce this removal of the Temp folder while MO2 is running through the MO2 GUI option **AND** through the Main Menu option by trying to install the Mod Configuration Menu for Fallout New Vegas. Every time it seems to remove the Temp folder. Therefore I think a note should be made about it on the wiki, but it is also a Quality of Life feature that SteamTinkerLaunch could have. FWIW it even seems like this is an issue on Windows though under different circumstances (the issue linked mentions a Reddit thread, and there are a few where users complain of needing to delete and recreate their Temp folder, or having their Temp folder disappear).

I was able to get MCM to install correctly by choosing the "Manual" option. There are some people complaining about a few mods however for different games using MO2 on Linux, see rockerbacon/modorganizer2-linux-installer#44. Maybe a note on the wiki about this mod specifically and the Temp folder disappearing in prefixes would suffice.

**Comment by frostworx** [comment]
Thanks for bringing this up. I wouldn't have noticed it at all, because I barely mod.
I'll add an automatic creation of the temp dir before MO2 is started, but as you wrote MO2 removed that dir while running, it would also make sense if you'd add a note in the wiki (thanks for the offer!).
Is there also an upstream issue. Sounds like it should be fixed there at the first place :)

edit: fun fact: just found that steamtinkerlaunch even has an option _(`CLEANPROTONTEMP`)_ to automatically remove that tempdir when closing steamtinkerlaunch

**Comment by frostworx** [comment]
I guess the above commit should already be enough, right? _(`Temp` dir in `steamuser` is automatically created directly before starting and after closing MO2)_

**Comment by sonic2kk** [comment]
Took a look at the commit and yup that looks like it should be perfect, thank you for the fast response, wow! :smile:

I'll add a note about it on the wiki. I noticed a pattern not specific to the mod I mentioned to reproduce the behaviour semi-reliably, which I'll make a note of on the [Modding page](https://github.com/frostworx/steamtinkerlaunch/wiki/Modding) as it has notes for another mod.

I bashed my head against the wall with this problem before when modding Oblivion and gave up so hopefully this change and wiki update info can help other users!

EDIT: Updated the wiki, I changed the format of the wiki page to hopefully divide up the sections a little bit better. My mod notes are under the [MO2 section](https://github.com/frostworx/steamtinkerlaunch/wiki/Modding#mo2). Feel free to modify anything that could be clarified.

**Comment by frostworx** [comment]
Thank you once more for another excellent wiki tune up! 👍

**Comment by Apacelus** [comment]
The instructions listed [here](https://github.com/frostworx/steamtinkerlaunch/wiki/Yad#compile-yad) may not work anymore(or at least they didnt for me). Please instead link to the [yad readme](https://github.com/v1cont/yad#readme) where the instructions are up2date.

Also, please let me create a small readme specifically for Ubuntu, as it was very frustrating to come to this project and find out that there is no native ubuntu version(which is understandable, I am not implying you have to make one) but also the instructions to install are scattered around the issues, the wiki, and some other places.
I am suggesting to make a separate Wiki page for Ubuntu and link to it in the readme, instead of having people figure it out themselves(after all they came here to quickly get a tool for a steam game, not to study the readme).

**Comment by frostworx** [comment]
Hi, 
thanks for the suggestion! You're right. A link to upstream instructions is definitely a better approach.
Also thanks for the offer to create a proper ubuntu wiki. Very appreciated!
I'm not 100% back from the current short trip _(already back, but still settling and already planning the next short trip after our upcoming "real" trip - yay, finally holidays)_, so please have some more patience. I'll update the yad wiki and also send you an invitation for updating the wiki soon.

**Comment by frostworx** [comment]
Just updated the "compile yad" section and sent you an invitation for collaboration, @Fornball
happy wiki boosting! :)

**Comment by Apacelus** [comment]
Hey, just added the [Ubuntu wiki entry](https://github.com/frostworx/steamtinkerlaunch/wiki/Ubuntu). I'm wondering, is there a FAQ/fixes page somewhere?

**Comment by frostworx** [comment]
Thanks for the Ubuntu wiki. I'll link to it in the main README.
No, there's no generic FAQ/fixes page yet.
All wikis where something special is worth to mention hopefully have a separate note (at least that was the initial idea once)
You're welcome to improve this of course anytime :)

Great to have you all onboard with improving the wiki! Thanks all :)

**Comment by zany130** [comment]
updated the wiki page for the dxvk hud menu to add a link to the DXVK readme here https://github.com/doitsujin/dxvk#hud= hope this was ok

EDIT: here is the link to the wiki page https://github.com/frostworx/steamtinkerlaunch/wiki/Dxvk-Hud-Options

**Comment by sonic2kk** [comment]
Think there was a slight typo in the link with the equals, but otherwise great! This keeps it consistent with other wiki pages :smile: 

It's not exclusive to this wiki page but maybe we should update the image on the DXVK Hud Options wiki, since the `0` option is gone now too. And also add a small note that having no selected options disables the DXVK hud (it wouldn't make sense to do it now since the image doesn't reflect the latest changes).

Also, the Gamescope wiki image could use an update. There might be other pages too but those are the ones I know off the top of my head :sweat_smile: I think those images are managed at a different repository though

**Comment by zany130** [comment]
yup yeah, the `=` at the end was messing up the link I just copied and pasted from my browser's URL which had that for some reason. and yeah seems like frostwork stores the images for the wiki and stuff here https://github.com/frostworx/repo-assets

**Comment by frostworx** [comment]
Thanks for the update @zany130
Yeah, there are multiple wikis already which have outdated pictures. I simply didn't find the time anymore to update them.
I moved all wiki assets into the separate repo-asset project to not force the users download them when pulling the main repo.
Of course I'm willing to invite everybody interested in updating wiki assets for collaboration in the repo-assets as well.

**Comment by HanPrower** [comment]
There should probably be a wiki page for the Flatpak now that it's live. Install/Uninstall/Caveats. Why there's no page for it on flathub.com (because their website doesn't list stuff for addons/extensions to runtimes :facepalm: )

I'd be happy to write it.

**Comment by frostworx** [comment]
Will need some days to read everything you did while I was away, but thanks a bunch already now for everything, @HanPrower , @sonic2kk @zany130 👍👍👍
@HanPrower, thanks for the offer, just added you as collaborator.

**Comment by HanPrower** [comment]
Flatpak page is done: https://github.com/frostworx/steamtinkerlaunch/wiki/Flatpak

Pretty sure it's clear and fairly concise.

**Comment by frostworx** [comment]
It is. Thank you very much!

**Comment by sonic2kk** [comment]
Off the back of #509 I updated the FSR wiki page: https://github.com/frostworx/steamtinkerlaunch/wiki/Wine-FSR

I took the opportunity to restructure the wiki page a little as well, adding in some extra information where I could. I also included the variable names for the two new options on their respective sections, to make it easier for users to check if their version of Wine/Proton supports it (by searching the documentation for it and/or trying the options manually with STL for debugging).

I tried my best to explain the options as best I could from my own understanding and providing relevant links. If there's anything that could be explained better, or that is unnecessarily explained, or anything else (even just typos :sweat_smile:) ping me and I can sort it. All feedback on the updated wiki is welcome :+1:

**Comment by zany130** [comment]
looks amazing to me. Nice 👍

EDIT: hope you don't mind I cleaned up a bit your gamescope page and added a warning for issue 490

**Comment by sonic2kk** [comment]
I looked through the revision diff for the GameScope wiki and it looks great, thank you! :+1: And adding the note for that issue was a good idea :-)

**Comment by sonic2kk** [comment]
To me, the pages for STL installation are a little "all over the place" - Not through anyone's fault, that's just how it progressed. When I started using STL there was really only package management installation and manual installation, which are mentioned on the wiki. Now, STL can be installed on Steam Deck, there are dedicated Ubuntu install instructions, and there are instructions for Flatpak Steam. STL has grown a lot very quickly :slightly_smiling_face: I have some ideas on how I feel installation instructions could be improved. It might be a big undertaking but I want to get some feedback on this, because after discussion if the ideas are considered good I'm happy to implement these changes in the wiki myself.

While the Readme does document all of these, it is a bit "messy", and I feel it could be beneficial to group installation instructions on the wiki together under a single Installation page. We could adapt the [existing Installation page](https://github.com/frostworx/steamtinkerlaunch/wiki/Installation) with headings for each method: **Package Management**, then breakdowns for package management *per-distro*, **Steam Deck**, **Flatpak Steam**, **Ubuntu**, **Manual Installation** and potentially **Non-Root Installation**. (see #548). The content of the existing "scattered" instructions would be copied over onto the wiki page.

Basically, I feel we should consolidate the installation instructions onto a single page. Ideally I would suggest we break them into separate pages under a category but [apparently GitHub doesn't allow for this](https://github.com/orgs/community/discussions/23914). So the next best thing in my opinion is to have one Installation page with the existing instructions moved into one page. This has a number of benefits:

1. A user doesn't have to look at the readme to find page links, they will all be under "Installation"
  a. This also means when a user searches the wiki for installation instructions, all they have to search for is "Install"
2. Using proper Markdown formatting, the user can still search for things like "Steam Deck" or "Flatpak".
3. Removes ambiguous wiki page titles, such as "Ubuntu". Maybe in future there will be Ubuntu-specific documentation, but right now the Ubuntu wiki page just has manual installation instructions.
4. Other pages such as Flatpak can simply link to the Flatpak heading on the Installation page, with something along the lines of "See Flatpak Installation Instructions". Likewise the instructions on this new wiki page on how to install on Flatpak should link back to the Flatpak page for caveats and more information. 
5. It becomes easier to add other types of installation instructions. Instead of expanding the Readme with more notes, we can just list the primary methods on the Readme, something along the lines of telling the user to use their package manager to install STL, and they can look or their distro in the table currently on the Readme. We can then tell them if their distro is not listed or STL is not up to date in their repositories, that they can refer to the Installation page for other installation methods. 

Sadly this has the problem of the people who wrote the original wiki pages losing their "credit" as the one who created the page. However, to mitigate this, we could just credit the contributors in some way when migrating installation instructions. The work gone into pioneering installation instructions for the likes of Ubuntu and Flatpak shouldn't go unnoticed imo. If anyone has any suggestions on this matter please give feedback!

I feel like this approach of having a single page for an installation guide helps clean up the documentation on the wiki and could help clean up the Readme too! All feedback on this is welcomed!

**The tl;dr here is to consolidate all the installation instructions under one, well formatted wiki page, to help new users find installation instructions more quickly and cleanly, to have one location to access all installation instructions, and to help maintain the wiki in future for other potential installation instructions!**

<hr>

One small aside I wanted to add here as it's slightly related: When we advertise Flatpak support, I feel like we should be clear that it _only_ refers to Flatpak Steam. I feel like the wiki page (simply titled "[Flatpak](https://github.com/frostworx/steamtinkerlaunch/wiki/Flatpak)") and the Readme don't make this clear enough. Thoughts on this? :slightly_smiling_face:

**Comment by zany130** [comment]
seems like a good idea to me! and I agree the flatpack thing should be clerified

**Comment by sonic2kk** [comment]
Thanks! I spent my evening attempting to improve a few wiki pages. Here is the main breakdown of my edits, feel free to compare diffs and suggest improvements :slightly_smiling_face: 
1. Updated [Flatpak wiki page](https://github.com/frostworx/steamtinkerlaunch/wiki/Steam-Flatpak) to explicitly reference Steam Flatpak.
2. Updated GameScope wiki page to include some [GameScope limitations and troubleshooting](https://github.com/frostworx/steamtinkerlaunch/wiki/GameScope#limitationsbugs), mainly things I have encountered myself such as a bug (maybe only on Wayland?) with the Meta+F shortcut causing problems. #490 also gets a mention here, as I've seen it come up a few times :smile: 
3. Made some initial improvements to the Installation wiki. I didn't add much of what I said I wanted to yet, I just made an initial effort to revamp the existing installation steps. I put the dependencies lists into tables, added links to the upstream projects and also expanded the list of dependencies to match the [AUR package](https://aur.archlinux.org/packages/steamtinkerlaunch-git?all_deps=1#pkgdeps).

All feedback on these changes are welcome. Hopefully these are some good improvements :sweat_smile: Presentation is everything especially for new users learning how to use SteamTinkerLaunch, and also I'm hoping for the installation section that this helps shine some light on just how much functionality STL has to offer - I didn't really know what all the optional dependencies were or what they did until tonight, and they seem pretty dang cool!

**Comment by sonic2kk** [comment]
Okay, I decided I wanted to make my fingers hurt and I went ahead and wrote up the Steam Deck/Steam Flatpak/Ubuntu+Similar install instructions.

In my initial post I said I wanted to simply copy the instructions over, however on reading them I felt I could break the informatin up and reformat it myself - no offense intended at all to the original authors! I just felt re-organizing some of the information would make it fit better with the style of the wiki and flow more naturally.

I made sure to credit the Steam Flatpak and Ubuntu authors for their information, and I haven't removed any information from the install Steam Deck/Steam Flatpak/Ubuntu wiki pages. I'd rather get some eyes on my changes and some more input before deciding to remove the install information on those pages. If these changes are good, the Ubuntu wiki page could probably be deleted as it only contains installation instructions.

I didn't add anything on non-root installation yet as I'd like some feedback on how that works and any associated caveats. The user in #548 didn't seem to have any problems but before I go ahead and add information on it to the wiki I'd like someone to weigh in :slightly_smiling_face: 

It's into the early morning where I am now, so it's possible I messed something up or missed something. All review on the wiki is welcome :smile: If and when the changes are considered good, I can get a PR up to update the parts of the Readme which refer to different install documents and have it all simply point to the new big Installation guide!

**Comment by sonic2kk** [comment]
Not strictly "wiki" related but I have a PR up for overhauling the Readme in a similar fashion to the Installation wiki: organizing information and expanding on certain points, in the hopes of making SteamTinkerLaunch more accessible to new users. See #552

**Comment by sonic2kk** [comment]
To help keep track of issues and what they're related to, I added a few labels and marked currently opened issues. Some issue discussion can get really lengthy and at first glance, it can be hard to tell what the core of an issue might be about. It also herlps keep issues organised, as you can click on labels and see all relevant open issues with that label.

If this isn't a good idea, we can remove the labels. I just thought it might be a helpful and relatively effortless change :slightly_smiling_face:

**Comment by frostworx** [comment]
Excellent idea. Thanks a lot for implementing it directly :)

**Comment by sonic2kk** [comment]
ProtonUp-Qt has just merged a SteamTinkerLaunch source. I have created a dedicated [ProtonUp-Qt wiki page](https://github.com/frostworx/steamtinkerlaunch/wiki/ProtonUp-Qt) (it won't let me put a hyphen in the page name :frowning_face:) with some general information and notes on how to get the most out of installing STL this way. Really the main notes are the benefits of installing via ProtonUp-Qt as well as how to use STL as a launch option this way (adding `$HOME/stl/prefix/steamtinkerlaunch %command%` instead of just being able to use `steamtinkerlaunch %command%` - ProtonUp-Qt always installs STL to the same path so this should always work).

As well as this I updated the [Installation wiki](https://github.com/frostworx/steamtinkerlaunch/wiki/Installation#protonup-qt) to make note of installing via ProtonUp-Qt.

Any feedback or changes are welcome, this is really just an initial submission to get something on the wiki about it to help guide users :smiley:

**Comment by sonic2kk** [comment]
GitHub's wiki features leave a lot to be desired imo, and some pages especially the Installation wiki page can be fairly detailed. GitHub does provide a table of contents for pages:

![image](https://user-images.githubusercontent.com/7917345/191585903-7379c469-39ae-4c75-99af-1c6308781ba7.png)

But if you go to a wiki page (say by [clicking here](https://github.com/frostworx/steamtinkerlaunch/wiki/Installation)), you'll notice the page you click on isn't automatically listed along the side, so you have to actually search for that wiki page to get the table of contents to show up! Kind of annoying.

For this reason I decided to generate and add a Table of Contents to the Installation wiki page. I used a tool called [Markdown-TOC](https://ecotrust-canada.github.io/markdown-toc/) which takes in your markdown text and generates a Table of Contents based on the headings structure. If anyone else wants to do this with other pages, you can just click the "Edit" button and paste it into the site, though I would recommend removing the first heading on the page which just has the page title, as it can kind of mess up the heading format (usually this is the first line of the page and only has one `#` beside it, e.g., `# Gamescope`).

This is probably only worth doing on some very wordy pages, and honestly IMO GitHub should make stuff like this more accessible so that the effort of markdown nerds doesn't go to waste. But I could write a book on how GitHub could improve their wiki feature, and for now I think this is a reasonable solution to help make it easier for users to find the information they need on long pages. If a user just wants to know how to install on Steam Deck they can see the heading link in the table of contents and click it.

I know it isn't ideal to have a hardcoded table of contents slapped to the top of the page, but GitHub doesn't allow you to auto-generate one and doesn't give a very intuitive way to view table of contents on wiki pages. They're able to do this for Readmes though, sigh.

**Comment by frostworx** [comment]
Nice idea! On the downside, it means more work for maintaining the wiki pages. Fortunately, there aren't that many pages where it could be useful _(I think you already picked the most important one 😀)_

**Comment by sonic2kk** [comment]
We've had quite a few people talking about using MO2 around here (which I think is pretty neat :smile:), so I took the opportunity to [expand and edit the MO2 wiki](https://github.com/frostworx/steamtinkerlaunch/wiki/Mod-Organizer-2) to have some more detailed information on usage and to hopefully clarify when to use Standalone Mode vs Game Mode, plus adding some info about being unable to mod native games with MO2 (ran into this issue twice earlier, it was a bit annoying).

I took out the references to how DotNet is used and installed as that's not really relevant anymore. I also added a note about the issues relating to the "Mod Manager" button as there is no ETA on when this will be fixed and a couple of users have had issues. Hopefully this combined with the info on the different MO2 modes helps prevent issues from running into issues.

As always feel free to give feedback or directly modify, or if it's not a good change just revert it entirely :smile: 

(*Also gave the [Winecfg wiki page](https://github.com/frostworx/steamtinkerlaunch/wiki/Winecfg) a little bit of love, as I linked to it on the MO2 page I wanted to make it a little prettier*)

<hr>

EDIT: Just updated the [Ubuntu wiki](https://github.com/frostworx/steamtinkerlaunch/wiki/Ubuntu-&-Derivitives) with some information about a critical breakage reported in #595. Ubuntu 20.04 ships a outdated version of glibc (2.31) and Yad versions now depend on glibc 2.34. I added a note to the top of the wiki to make users aware of this.

**Comment by sonic2kk** [comment]
Since #608 was merged, I thought it would be nice to update the installation wiki with an [uninstall section](https://github.com/frostworx/steamtinkerlaunch/wiki/Installation#uninstall), since now SteamTinkerLaunch can be uninstalled in a very straightforward way for all installation methods :tada:

**Comment by sonic2kk** [comment]
Hey everyone!

I was having a look around the wiki as I usually do late at night and I was just thinking that the wiki has some pages that could be potentially consolidated into one page, such as [Compatdata](https://github.com/frostworx/steamtinkerlaunch/wiki/Compatdata), [Compatdata Delete](https://github.com/frostworx/steamtinkerlaunch/wiki/Compatdata-Delete) and [Compatdata Redirect](https://github.com/frostworx/steamtinkerlaunch/wiki/Compatdata-Redirect). I'm not saying the information on these pages has to change by any means, but maybe it would be a bit cleaner to have one "Compatdata" document?

Wikis can be daunting and having fewer pages to sift through could help new users get to grips with using SteamTinkerLaunch. Of course it's important to be mindful of when this can become "too much" and sometimes it's beneficial to have separate documents - I'm currently considering this for the Installation wiki and wondering if it's better to have a separate uninstall doc, but that's for another time :-) 

There's also a potential issue of this interfering with the wiki shortcut from inside SteamTinkerLaunch. If this kind of change would break any functionality on that end then this idea is probably not worth it.

**Comment by frostworx** [comment]
Wiki shortcuts should stay intact of course, but otherwise I wouldn't mind a merge of wikis where it makes sense. 👍

**Comment by AliceDTRH** [comment]
I don't know how the wiki shortcuts are setup in the application, but you can just link to parts of pages like this:
``https://github.com/frostworx/steamtinkerlaunch/wiki#gui`` 
so in this example it would become:
 ``https://github.com/frostworx/steamtinkerlaunch/wiki/Compatdata#Delete`` for example.

I only just realized the size of this issue, I haven't read the other comments so I'm sorry if this was already mentioned.

**Comment by frostworx** [comment]
true.
The corresponding gui function would simply require a new or updated `CURWIKI` export.
_(example: `export CURWIKI="$PPW/Category-Menus"`)_
I'd guess "# chapters" should just work here as well.

**Comment by sonic2kk** [comment]
Added a sidebar to the wiki (just learned about that feature recently!) which includes some quick links to what I think are useful wiki pages:

![image](https://user-images.githubusercontent.com/7917345/194963468-25c07eed-18b0-4161-84ab-fd2935b1a98b.png)

Feel free to edit this to add/remove/change pages :smiley:

**Comment by sonic2kk** [comment]
Gave the Steam Deck wiki a pretty huge overhaul , feedback welcome: https://github.com/frostworx/steamtinkerlaunch/wiki/Steam-Deck

**Comment by frostworx** [comment]
Wow, thank you very much! Great re-write!

**Comment by sonic2kk** [comment]
Added a section to the Modding wiki page on how to install the [Resident Evil 4 HD mod](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Modding#resident-evil-4-hd) (it's basically just running the setup exe as a custom command and then setting a Wine DLL override using STL)

**Comment by sonic2kk** [comment]
I've been slacking recently on the wiki updates (except for the changelog) but today I tried to update the wiki a little:
- Updated GameScope wiki page to reflect that it now uses NIS v1.0.3, and removed the mention of Cheat Engine as it is no longer supported: https://github.com/sonic2kk/steamtinkerlaunch/wiki/GameScope
- Add an FAQ wiki page to address some of the questions I've seen: https://github.com/sonic2kk/steamtinkerlaunch/wiki/FAQ

**Comment by sonic2kk** [comment]
One-Time Run got a bit of love recently and so I updated the wiki page for it: https://github.com/sonic2kk/steamtinkerlaunch/wiki/One-Time-Run

The entire page was rewritten pretty much and an updated screenshot was included.

**Comment by sonic2kk** [comment]
Hard dependencies were updated to include `make` as a hard dependency, as per #746.

**Comment by sonic2kk** [comment]
Added a "Paths" section to the wiki sidebar as suggested by @zany130 in #762 - Thanks for suggesting it, makes sense to me :-) 

It could still use a better name but I'll look into that in future.

**Comment by sonic2kk** [comment]
Updated the [Paths wiki page](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Paths) to include a more comprehensive list of paths, and use tables to group common paths together. Could probably still be improved and I am sure I have missed some paths, but I tried to be more comprehensive.

**Comment by sonic2kk** [comment]
Seems like only I used this issue, and I have not used it to document the other wiki changes I have made in a while, so I will close this.

The wiki is still a WIP, but I think dedicated issues for it is the better approach these days :-)
